tests/integration/NewLexemeForm.test.ts

```diff
@@ -235,6 +235,10 @@ describe( 'NewLexemeForm', () => {
 			await wrapper.trigger( 'submit' );
 
 			expect( createLexeme ).toHaveBeenCalledWith( 'foo', 'de', 'Q123', 'Q456' );
+
+			await nextTick();
+			await nextTick();
```

Comment:
Not sure if that should also be `flushPromises()`

Reply:
probably… if this is between expectations on `createLexeme` and `goToTitle`, it’s not really about DOM modification, right?